src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const Input = (props) => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    ref.current.focus();
+    ref.current.addEventListener("blur", function () {
+      props.onEditComplete(ref.current.value);
+    });

```

Comment:
```suggestion
    ref.current.addEventListener("blur", () => {
      props.onEditComplete(ref.current.value);
    });
```
特別な理由がないかぎりアロー関数を使うことが推奨されているので、可能であれば揃えた方がいい気がします！